english/youre-fine/youre-fine.md

```diff
@@ -1,3 +1,4 @@
 Whew, you're fine. It was just chocolate.
 
 [Back to the room...](../marshmallow.md)
+[Relax with a haiku...](../hiku.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.